kite-data/kite-data-oozie/src/main/java/org/kitesdk/data/oozie/KiteConfigurationService.java

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright 2015 Cloudera Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kitesdk.data.oozie;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.oozie.ErrorCode;
+import org.apache.oozie.service.HadoopAccessorException;
+import org.apache.oozie.service.HadoopAccessorService;
+import org.apache.oozie.service.Service;
+import org.apache.oozie.service.ServiceException;
+import org.apache.oozie.service.Services;
+import org.apache.oozie.util.XConfiguration;
+import org.apache.oozie.util.XLog;
+
+public class KiteConfigurationService implements Service {
+
+  public static final String CONF_PREFIX = Service.CONF_PREFIX + "KiteConfigurationService.";
+  public static final String KITE_CONFIGURATION = CONF_PREFIX + "kite.configuration";
+
+  private static XLog LOG = XLog.getLog(KiteConfigurationService.class);
+  private Configuration conf;
+  private Configuration kiteConf;
+
+  @Override
+  public void init(Services services) throws ServiceException {
+    conf = services.getConf();
+    try {
+      loadKiteConf(services);
+    } catch(IOException ioe) {
+        throw new ServiceException(ErrorCode.E0100, KiteConfigurationService.class.getName(), "An exception occured while attempting"
+                + "to load the Kite Configuration", ioe);
+    }
+  }
+
+  public Configuration getKiteConf() {
+    return kiteConf;
+  }
+
+  // largely borrowed with from the HCatAccessorService
+  // configuration methodology. Modification includes the
+  // ability to specify multiple paths to configurations
+  private void loadKiteConf(Services services) throws IOException {
+    String[] paths = conf.getStrings(KITE_CONFIGURATION);
```

Comment:
Is this static reference something added in a more recent Oozie version? oozie-core-4.1.0 doesn't seem to make this available. I could switch this off to "services.getConf().getStrings(KITE_CONFIGURATION)" though which would help to avoid storing off the configuration reference.